playground/nextjs/src/components/Header.tsx

```diff
@@ -0,0 +1,36 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import React from 'react';
+
+const LINKS = [
+  { href: '/', label: 'Default Inbox' },
+  { href: '/render-bell', label: 'Render Bell' },
+  { href: '/render-notification', label: 'Render Notification' },
+  { href: '/preferences', label: 'Preferences' },
+  { href: '/notifications', label: 'Notifications' },
+];
+
+const NavLink = ({ href, label }: { href: string; label: string }) => {
+  const router = useRouter();
+
+  const pathname = router.pathname;
+
+  const isActive = pathname === href;
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
✏ **suggestion:** Looks like `usePathname` from `next/navigation` may be the [recommended approach](https://blog.logrocket.com/how-to-use-type-guards-typescript/#:~:text=a%20Boolean%20value.-,Custom%20type%20guards,-TypeScript%20custom%20type)‏

Reply:
I think it is limited to `app router` only.
The playground here uses the `pages router`

Reply:
We will be adding a folder for app router as well, so we'll be using it there!